docs/modules/ROOT/pages/index.adoc

```diff
@@ -242,14 +243,31 @@ The content of `signature` depends on the type of credential that is used. Valid
 
-[#describe-role-in-content]
-NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/14[issue #14]): Describe how to allow actors to describe their role in the content (previously described via CreativeWork assertion) and potentially in specific steps referenced in C2PA actions assertion.
-
 [#describe-social-media]
 NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/15[issue #15]): Describe where would things like social media accounts live.
 
-[#describe-actor-role]
-NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/26[issue #26]): Describe credential holder’s role in relation to the asset.
+==== Credential subject roles
+
+The _<<_credential_holder,credential holder>>_ MAY choose to describe the relationship of the _<<_credential_subject,credential subject>>_ to the _<<C2PA asset>>_ by adding a `role` field to `signer_payload`.
+
+The following values for `role` are defined. It is recommended that the value of `role` be one of the following values and that any user experience that presents the content of an *<<_identity_assertion,identity assertion>>* be prepared to describe these roles in the same human language as the surrounding interface:
+
+[width="100%",cols="4,10",options="header"]
+|=======================
+| Value        |  Meaning
+| `cawg.creator` | The _<<_credential_subject,credential subject>>_ is a primary creator/author of this _<<C2PA asset>>._
+| `cawg.contributor` | The _<<_credential_subject,credential subject>>_ is a secondary creator/author of this _<<C2PA asset>>._
+| `cawg.editor` | The _<<_credential_subject,credential subject>>_ is an editor of this _<<C2PA asset>>._
+| `cawg.producer` | The _<<_credential_subject,credential subject>>_ is a producer of this _<<C2PA asset>>._
+| `cawg.publisher` | The _<<_credential_subject,credential subject>>_ is a publisher of this _<<C2PA asset>>._
+| `cawg.reviewer` | The _<<_credential_subject,credential subject>>_ is a reviewer of this _<<C2PA asset>>._
+| `cawg.sponsor` | The _<<_credential_subject,credential subject>>_ has supported or sponsored the creation of this _<<C2PA asset>>._
+| `cawg.translator` | The _<<_credential_subject,credential subject>>_ has adapted of this _<<C2PA asset>>_ from a similar work in another language.
+|=======================
+
+Other values for `role` MAY be used with the understanding that they may not be well understood by *<<_identity_assertion,identity assertion>>* consumers. Values for `role` that begin with the prefix `cawg.` are reserved for the use of the Creator Assertions Working Group and MUST NOT be used unless defined in a this or a future version of this specification.
```

Comment:
Should you point at the relevant section of C2PA for namespacing rules. so that all other roles are namespaced??

Reply:
@lrosenthol can you point me to that section? IIRC when we discussed a few weeks ago, that wasn't yet formalized.

Reply:
@lrosenthol the closest section in C2PA that I could find is [§6.2 Labels](https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_labels), which appears to be specific to _assertion_ labels and thus not usable here. Is there another section I'm missing?